src/sentry/backup/imports.py

```diff
@@ -189,7 +190,7 @@ def _import(
                 del content_as_json[i]
 
         # Return the content to byte form, as that is what the Django deserializer expects.
-        content = json.dumps_experimental("backup.enable-orjson", content_as_json)
+        content = orjson.dumps(content_as_json).decode()
```

Comment:
If your change is correct, than previous comment needs to be changed:

```suggestion
        content = orjson.dumps(content_as_json)
```